OpenSim/Common/Component.cpp

```diff
@@ -134,11 +152,19 @@ void Component::finalizeFromProperties()
 }
 
 // Base class implementation of virtual method.
-// Call extendFinalizeFromProperties on all components
+// Call finalizeFromProperties on all subcomponents
 void Component::componentsFinalizeFromProperties() const
 {
-    for (unsigned int i = 0; i<_components.size(); i++){
-        _components[i]->finalizeFromProperties();
```

Comment:
`_propertySubcomponents` is an array of `SimTK::ReferencePtr`s and `->` operator returns a writeable reference even though it is a const method. The other two arrays use `SimTK::ClonePtr` and here `get()` is const and returns const reference. `componentsFinalizeFromProperties` itself is const to reflect that `*this` is not being modified and `finalizeFromProperties` is being called on its subcomponents. I am on the fence on whether that is important/relevant distinction or not.